cpu/schbench.py

```diff
@@ -22,9 +23,26 @@
 from avocado.utils import build, distro, git
 from avocado.utils.software_manager.manager import SoftwareManager
 
+DEFAULT_SCHBENCH_URL = 'https://git.kernel.org/pub/scm/linux/kernel/git/mason/schbench.git'
```

Comment:
can this be a local variable like in SETUP function itself

Reply:
Thanks for the suggestion, removed this, instead of having an additional constant to track this kept it as a part of `self.params.get(.., defaut=**)` similar to other tests.